wasm/module.go

```diff
@@ -461,3 +466,42 @@ func (m *Module) allDeclarations() (functions []Index, globals []*GlobalType, me
 	tables = append(tables, m.TableSection...)
 	return
 }
+
+// SectionSize returns the count of items in a given section
+func (m *Module) SectionSize(sectionID SectionID) uint32 {
+	switch sectionID {
+	case SectionIDCustom:
+		count := uint32(len(m.CustomSections))
+		if m.NameSection != nil {
```

Comment:
this seems inconsistent to others, as we have multiple custom sections vs unique one for others, and the comment `// SectionSize returns the count of items in a given section` is not true for custom section, right?

Reply:
you are right, that is indeed a special case!

we can do one of two things I think.

* Change the wording of that so that it is true even in the special case: `// SectionSize returns the count of items for a given section ID`
* Exclude CustomSection and then the docs around it

I feel the former is less to explain wdyt?

Reply:
what confuses me is that there's no "items" for custom section ID, but there's multiple "sections" for custom section ID. 
`// SectionSize returns the count of items for a given section ID except that if the id is custom section id, returns the number of custom sections` would wok for me

Reply:
The spec itself is the confusing thing. I addressed this part and gave examples of "items" instead. How about it?

```
// SectionSize returns the count of items for a given section ID
//
// For example, given...
// * SectionIDType this returns the count of FunctionType
// * SectionIDCustom this returns the count of unique section names
// * SectionIDExport this returns the count of unique export names
```

Reply:
sg